### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by ShingiraiBhengesa**
**Loom Video**
https://www.loom.com/share/307ed52c982e47aebd11f963efc5c455

**Created the Air Pollution Index With The Following features:**

- [ ] Navigation between two pages, home and contamination details.
- [ ] Users can start a search by entering the city name in the search bar on the home page, which will display a list of matching cities.
- [ ] Users can view the detailed information on air pollution by clicking the arrow button located near to each city.
- [ ] The details page shows:
- [ ] The air quality index information using international standard colors.
- [ ] The city location information.
- [ ] The forecast of air pollution for the next 24 hours.
- [ ] The forecast of air pollution for the next 5 days.
- [ ] A table with the measurement of each polluting substance.
- [ ] The user can return to the home page by clicking the back button.
- [ ] The website retrieves city and air pollution data from two different third-party APIs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,50 @@
...
-- Frameworks
-- Technologies used
+- React
+- Redux
+- SCSS
 
-## Live Demo (if available)
+## Live Demo
 
 [Live Demo Link](https://livedemo.com)
```
Comment (keroloussamy) [inline README.md]: - [ ] Kindly add the link for your deployed project.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,50 @@
...
+👤 **Shingirai**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@ShingiraiBhengesa](https://github.com/ShingiraiBhengesa)
+- Twitter: [@bhengesa](https://twitter.com/twitterhandle)
+- LinkedIn: [Shingirai Bhengesa](https://www.linkedin.com/in/shingirai-bhengesa-612b09206/)
 
 ## 🤝 Contributing
```
Comment (keroloussamy) [inline README.md:48]: - [ ] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @ShingiraiBhengesa, 

Good job so far! Your project is really amazing
There are some small issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,56 +1,45 @@
...
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
```
Comment (medaminedev66) [inline README.md:17]: - [ ] In your Readme, there is no live demo link. Following the project requirements, your application should be deployed using Heroku, Netlify, or GitHub Pages, and a link to the deployed project should be added to your README.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ShingiraiBhengesa 😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Your design looks great
- Your app works as expected and runs without errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ShingiraiBhengesa ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**